pages/common/gammastep.md

```diff
@@ -0,0 +1,28 @@
+# Gammastep
+
+> Adjust the color temperature of your screen according to the time of day.
+> More information: <https://gitlab.com/chinstrap/gammastep>.
+
+- Turn on Gammastep with 5700K [t]emperature during the day and 3600K at night:
+
+`gammastep -t {{5700}}:{{3600}}`
```

Comment:
Imo it should still be a placeholder because placeholders mark the part that can be changed by the user. Instead we should change the description.

Reply:
I certainly intended for the placeholders to be placeholders rather than explicit examples. One moment for an update...

Reply:
I also applied this suggestion to screen brightness, although that edit makes the value range description more cumbersome. 

It raises a new question: is it better to be more concise (i.e. not specify value ranges) than to include the generic placeholder -- or instead better to replace specific placeholder examples with generics?

Reply:
> It raises a new question: is it better to be more concise (i.e. not specify value ranges) than to specify the values of the generic placeholder -- or instead better to replace specific examples with generic placeholders?

Imo that's a question of taste. Afaik the tldr project does not have a policy on that. I personally think that it is better to have specific examples (such as this page) because this automatically resolves questions about e.g. the required data type, format etc.

> I ask partially because screen brightness now seems longer than other command descriptions I've used in TLDR; and partially because this PR is setting up a general review of color-shifting commands. The suggestions from this review increase divergence between gammastep and redshift

I'll change the redshift page. I personally think that the descriptions are not too long.

Reply:
@sebastiaanspeck - what do you think? I believe your review is the last element of this PR.